src/mono/netcore/Makefile

```diff
@@ -70,18 +58,6 @@ run-tests-coreclr-all:
 #   make run-tests-corefx-System.Runtime 
 # for `System.Runtime.Tests` (the rule builds both the lib and the test if needed)
 run-tests-corefx-%:
-	$(DOTNET) msbuild /t:RunLibrariesTest /p:LibraryToTest=$* $(MONO_PROJ)
-
-# precompile (AOT) all libraries' tests
-aot-tests-corefx:
-	$(DOTNET) msbuild /t:PrecompileLibrariesTests $(MONO_PROJ)
-
-# precompile (AOT) all libraries inside testhost (it's used for libraries' tests)
-aot-testhost-corefx:
-	$(DOTNET) msbuild /t:PrecompileTesthostLibraries $(MONO_PROJ)
+	$(DOTNET) msbuild /t:PatchLibrariesTesthost $(MONO_PROJ)
+	$(DOTNET) build ../../../src/libraries/$*/tests -c $(LIBRARIES_TESTS_CONFIG) /p:RuntimeFlavor=mono /t:Test
```

Comment:
Why isn't this just `$(TOP)/build.sh -s mono+libs+libs.tests  -rc $(RUNTIME_CONFIG) -c $(LIBRARIES_TESTS_CONFIG) /p:RuntimeFlavor=mono /t:Test` or something like that?  I thought we didn't need to do patching by hand anymore

Reply:
The desktop build should both generate an in-tree runtime pack and a testhost.  Unless I'm missing something, the `PatchLibrariesTesthost` target shouldn't be necessary.

Reply:
Yeah makes sense to drop it.